services/cargohold/src/CargoHold/Options.hs

```diff
@@ -104,7 +104,7 @@ data AWSOpts = AWSOpts
     --
     -- This logic is: If the @Z-Host@ header is provided and found in this map,
     -- the map's values is taken as s3 download endpoint to redirect to;
-    -- otherwise, `_awsS3DownloadEndpoint` is used. This option is only useful
+    -- otherwise an 302 error is thrown. This option is only useful
```

Comment:
```suggestion
    -- otherwise an 404 error is thrown. This option is only useful
```

Thanks for adjusting this outdated doc!

However, I think a `NoMatchingAssetEndpoint` is thrown in this case (map defined, but host not found in it.); which translates to a `404`. :thinking:

Reply:
The `302` you may haven seen in e.g. tests is likely a redirect to the assets host (e.g. AWS S3, MinIO, ...)

Reply:
Ouch, yes of course. 302 is als not an error.